platform-hub-api/app/controllers/me_controller.rb

```diff
@@ -40,6 +42,24 @@ def complete_hub_onboarding
     end
   end
 
+  def complete_services_onboarding
+    # Currently only supports GitHub onboarding
+    success = handle_onboard_github_request current_user, audit_context
+
+    current_user.update_flag :completed_services_onboarding, true
+    current_user.save  # Note: doesn't _really_ matter if this save fails – just means the flag won't get set
```

Comment:
Yeah, exactly – the worse case is we have a bug and flag triggers the banner in the UI permanently (which we'll add in a separate PR), which would be annoying, but not critical.